### PR TITLE
Ht20 247 asset listener

### PR DESCRIPTION
**Pull request by anefiox**
New listener for asset posts

Tests for new listener

### CODE REVIEW

**Review by rcmsoftwaredevelopment — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -348,3 +348,6 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+/docker-compose.override.yml
```
Comment (humulla) [inline .gitignore:351]: What is this used for?

File: .gitignore [hunk 2]
```diff
@@ -348,3 +348,6 @@ MigrationBackup/
...
+/docker-compose.dcproj
```
Comment (humulla) [inline .gitignore:352]: What is this used for?

File: HousingSearchListener/V1/Domain/Asset/Asset.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
+using System;
+
+namespace HousingSearchListener.V1.Domain.Asset
+{
+    public class Asset
```
Comment (humulla) [inline HousingSearchListener/V1/Domain/Asset/Asset.cs:5]: Can we use the Shared Asset nuget package instead

File: HousingSearchListener/V1/Domain/Asset/AssetAddress.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
+namespace HousingSearchListener.V1.Domain.Asset
+{
+    public class AssetAddress
```
Comment (humulla) [inline HousingSearchListener/V1/Domain/Asset/AssetAddress.cs:3]: Can we use the shared nuget package instead 
https://github.com/LBHackney-IT/asset-shared

File: HousingSearchListener/V1/Domain/Asset/AssetCharacteristics.cs [hunk 5]
```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace HousingSearchListener.V1.Domain.Asset
+{
+    public class AssetCharacteristics
```
Comment (humulla) [inline HousingSearchListener/V1/Domain/Asset/AssetCharacteristics.cs:5]: Use Shared Nuget package

File: HousingSearchListener/V1/Domain/Asset/AssetLocation.cs [hunk 6]
```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+
+namespace HousingSearchListener.V1.Domain.Asset
+{
+    public class AssetLocation
```
Comment (humulla) [inline HousingSearchListener/V1/Domain/Asset/AssetLocation.cs:5]: Use Shared Nuget Package

File: HousingSearchListener/V1/Domain/Asset/AssetManagement.cs [hunk 7]
```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace HousingSearchListener.V1.Domain.Asset
+{
+    public class AssetManagement
```
Comment (humulla) [inline HousingSearchListener/V1/Domain/Asset/AssetManagement.cs:5]: Use Shared Nuget Package

File: HousingSearchListener/V1/Domain/Asset/AssetTenure.cs [hunk 8]
```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Text.Json.Serialization;
+using Hackney.Shared.Tenure.Domain;
+
+namespace HousingSearchListener.V1.Domain.Asset
+{
+    public class AssetTenure
```
Comment (humulla) [inline HousingSearchListener/V1/Domain/Asset/AssetTenure.cs:7]: Use Shared Nuget Package

File: HousingSearchListener/V1/Domain/Asset/Enums.cs [hunk 9]
```diff
@@ -0,0 +1,30 @@
+using System.Text.Json.Serialization;
+
+namespace HousingSearchListener.V1.Domain.Asset
+{
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum AssetType
```
Comment (humulla) [inline HousingSearchListener/V1/Domain/Asset/Enums.cs:6]: Use Shared Nuget Package

File: HousingSearchListener/V1/Domain/Asset/ParentAsset.cs [hunk 10]
```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace HousingSearchListener.V1.Domain.Asset
+{
+    public class ParentAsset
```
Comment (humulla) [inline HousingSearchListener/V1/Domain/Asset/ParentAsset.cs:5]: Use Shared Nuget Package

**Pull request merged**